src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/hooks/useProgramMetadata.js

```diff
@@ -8,7 +8,7 @@ const query = {
         id: ({ id }) => id,
         params: {
             fields:
-            ['programStages[id,repeatable,hideDueDate,programStageDataElements[displayInReports,dataElement[id,valueType,displayName,displayFormName,optionSet[options[code,name]]]'],
+            ['programStages[id,repeatable,hideDueDate,enableUserAssignment,programStageDataElements[displayInReports,dataElement[id,valueType,displayName,displayFormName,optionSet[options[code,name]]]'],
```

Comment:
Do you know the reason why `useProgramMetadata` is retrieving data from the api instead of using the IndexedDB cache? If not, can you change it to use `useIndexedDBQuery`?

Reply:
I'm not sure, but it could perhaps be related to translations, i.e. `displayName` needs to come from the server?

Reply:
We have a ticket to fix the displayName cache problem. Great if you change this one to use `useIndexedDBQuery`

Reply:
If I understand you correctly, there is a separate store/table for data elements. Should be retrieved from there.

Reply:
Changed the metadata source in this case from the server to IndexedDB. The code got rather complicated the way I did it though. Got any suggestions for making it less complicated?

Reply:
It would be beneficial to have one approach for this and currently that is `useIndexedDBQuery`. If we don't like the approach there, we can look into what we don't like and change it for all queries to IndexedDB. But let's start by using `useIndexedDBQuery`.